### DISCUSSION TITLE
Full Day Event Error

### DISCUSSION THREAD

**Discussion by overunderspace** [MMRIZE/MMM-CalendarExt3Journal#6] [General]
When there is a single full day event in the future beyond the days shows on the module, it will show up currently and as if it is covering every day. I put a test full day event on June 21st (over 3 weeks away) but it is showing up on my module set for all the days in the module. 
![image](https://github.com/MMRIZE/MMM-CalendarExt3Journal/assets/99686317/e12e0032-f1bc-4dfe-9a84-2462c702e2ac)

**Comment by eouia** [comment]
Can you share your system timezone information(where are you living) and configJS and ics url/file with me? (eouia0819@gmail.com) 
Without real data, i cannot guess.

**Reply by overunderspace** [reply]
I sent you the email. Is it possible to use eventTransform to hide any fullday/multiday events if they are over a certain amount of days in the future?

**Reply by eouia** [reply]
I'm looking inside your config. Sorry for late. Plz wait a while.

**Reply by eouia** [reply]
I updated. Plz update your module and test it.

**Reply by overunderspace** [reply]
That fixed it, thank you so much! I am loving the module!
